ding/worker/collector/base_serial_evaluator_ngu.py

```diff
@@ -192,18 +192,24 @@ def eval(
         self._policy.reset()
         beta_index = {i: 0 for i in range(self._env_num)}  # TODO
         beta_index = to_tensor(beta_index, dtype=torch.int64)
-        prev_action = {i: torch.tensor(-1) for i in range(self._env_num)}  # TODO    self.action_shape
+        prev_action = {i: torch.tensor(-1) for i in range(self._env_num)}  # TODO: self.action_shape
         prev_reward_e = {i: to_tensor(0, dtype=torch.float32) for i in range(self._env_num)}
         with self._timer:
             while not eval_monitor.is_finished():
                 obs = self._env.ready_obs
                 obs = to_tensor(obs, dtype=torch.float32)
-                # policy_output = self._policy.forward(obs,beta_index)
-                policy_output = self._policy.forward(beta_index, obs, prev_action, prev_reward_e)  # TODO action,r_e,r_i
+                # TODO(pu): r_i, reward embeding
+                policy_output = self._policy.forward(beta_index, obs, prev_action, prev_reward_e)
```

Comment:
why not `beta_index` belongs to policy, rather than in evaluator/collector, it is not modified in `evaluator/collector`